Dockerfile

```diff
@@ -80,6 +84,7 @@ FROM base AS dynamic-link
 
 COPY --from=dynamic-build /opt /opt
 
+LABEL br.lnls.epics-in-docker.build-target="dynamic-link"
```

Comment:
Maybe simply `target` is enough? There is no such thing as `runtime-target`, for instance.